spec/requests/providers/property_value_spec.rb

```diff
@@ -62,7 +62,6 @@
           context 'property is owned outright' do
             let(:legal_aid_application) { create :legal_aid_application, :with_applicant, :with_own_home_owned_outright }
 
-            # TODO: replace with correct path once other controllers are ready
```

Comment:
Wasn't sure about this one. It appears to be a bit superfluous, mainly because if there was a change, it sounds like this test would break and need changing anyway.

Reply:
This looks fine to delete. The test below has been updated 12 times so I'd assume that, in one of those, the URL was updated 🤷‍♂️